line_profiler/line_profiler.py

```diff
@@ -155,6 +155,8 @@ def add_module(self, mod):
         return nfuncsadded
 
 
+# This could be in the ipython_extension submodule,
+# but it doesn't depend on the IPython module so it's easier to just let it stay here.
```

Comment:
I agree, this is just handling the case where we are printing something in a notebook. It doesn't have to do with the plugin directly, so it's fine to stay here.